src/Lesson3/Task.java

```diff
@@ -0,0 +1,34 @@
+package Lesson3;
+
+import java.util.Scanner;
+
+public class Task {
+    public static void main(String args[]) {
+        int i, num, num1;
```

Comment:
It's better to inline declaration and initialization of these variables and place it as closer as possible to the place of use